### PR TITLE
Test migration scrips for pool comments

### PR DESCRIPTION
**Pull request by shankar-pankhania-hmcts**

### CODE REVIEW

File: Postgres old schema to new schema migration/0026 - pool_comments.sql [hunk 1]
```diff
@@ -64,15 +64,15 @@ alter table juror_mod.pool_comments
...
-	
+--exception
+--	when others then
+--		update	juror_mod.migration_log
+--		set		"status" = 'ERROR',
+--				actual_target_count = 0,
+--				end_time = now(),
+--				execution_time = age(now(), migration_log.start_time)
+--		where 	script_number = '0026';
+--
```
Comment (CCHAPMAN01) [inline Postgres old schema to new schema migration/0026 - pool_comments.sql:75]: please reinstate this code block

File: Postgres old schema to new schema migration/test/0026 - pool comments/0026 - pool comments test setup.sql [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+insert into juror.unique_pool(pool_no, "owner", return_date, no_requested, pool_type, loc_code, new_request, next_date, reg_spc) VALUES 
+('415220502', '415', '2022-05-03', 5, 'CRO', '415', 'N', current_date, 'R'),
+('415220503', '416', '2022-05-03', 6, 'CRO', '416', 'N', current_date, 'R'),
+('415220504', '417', '2022-05-03', 7, 'CRO', '417', 'N', current_date, 'R');
+
+INSERT INTO juror.pool_comments ("owner", pool_no, user_id, last_update, pcomment, no_requested) VALUES
+('415', '415220502', '1', '2023-01-01', 'test1', 1),
+('416', '415220503', '2', '2023-01-01', 'test2', 2),
+('417', '415220504', '3', '2023-01-01', 'test3', 3);
```
Comment (CCHAPMAN01) [inline Postgres old schema to new schema migration/test/0026 - pool comments/0026 - pool comments test setup.sql:12]: please add a test case where no_requested is null (test the default value of 0 is set)

Comment (CCHAPMAN01) [inline Postgres old schema to new schema migration/test/0026 - pool comments/0026 - pool comments test setup.sql:12]: Please add a test case where last_update is null

**Review by CCHAPMAN01 — CHANGES_REQUESTED** [review]
Please see review comments

File: Postgres old schema to new schema migration/test/0026 - pool comments/0026 - pool comments test verification.sql [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    user_id varchar(20) not null,
+    last_update timestamp(0) null,
+    pcomment varchar(80) not null,
+    no_requested int4 null);
+
+insert into temp_pool_comments (pool_no,user_id,last_update,pcomment,no_requested) values
+('415220502', '1', '2023-01-01', 'test1', 1),
+('415220503', '2', '2023-01-01', 'test2', 2),
+('415220504', '3', '2023-01-01', 'test3', 3),
+('415220505', '4', '2023-01-01', 'test4', 0),
```
Comment (CCHAPMAN01) [inline Postgres old schema to new schema migration/test/0026 - pool comments/0026 - pool comments test verification.sql:12]: The no_requested value migrated as null - this looks to be correct as the column allows nulls, please update the expected value to be null so the verification scripts do not return any rows

**Review by CCHAPMAN01 — APPROVED** [review]
